### PR TITLE
Release Parity v2.2.0

### PR DESCRIPTION
**Pull request by geoffharcourt**

### CODE REVIEW

File: Formula/parity.rb [hunk 1]
```diff
@@ -3,9 +3,9 @@
 class Parity < Formula
   homepage "https://github.com/thoughtbot/parity"
   head "https://github.com/thoughtbot/parity.git"
-  sha256 "240b96b441cda64c17a433dde0f6c3e6f6381ed13754bbf844f47028b8d254e9"
-  url "https://github.com/thoughtbot/parity/releases/download/v2.1.0/parity-2.1.0.tar.gz"
-  version "2.1.0"
+  sha256 "e9381fd8ee3d8036d9843198584cdf742ee4470af26a19eba45eaeb5f4ec39eb"
+  url "https://github.com/thoughtbot/parity/releases/download/v2.2.0/parity-2.2.0.tar.gz"
```
Comment (geoffharcourt) [inline Formula/parity.rb]: Now that we're not doing any packaging weirdness, it's occurred to me that we can stop uploading a tarball and just use GH's default bundle of the source code...

**Comment by mike-burns** [comment]
```
~% curl -LO https://github.com/thoughtbot/parity/releases/download/v2.2.0/parity-2.2.0.tar.gz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   595    0   595    0     0   2290      0 --:--:-- --:--:-- --:--:--  2297
  8  198k    8 16360    0     0  18692      0  0:00:10 --:--:--  0:00:10 18692^[100  198k  100  198k    0     0   141k      0  0:00:01  0:00:01 --:--:--  342k
~% sha256sum parity-2.2.0.tar.gz
e9381fd8ee3d8036d9843198584cdf742ee4470af26a19eba45eaeb5f4ec39eb  parity-2.2.0.tar.gz
~% [ e9381fd8ee3d8036d9843198584cdf742ee4470af26a19eba45eaeb5f4ec39eb = e9381fd8ee3d8036d9843198584cdf742ee4470af26a19eba45eaeb5f4ec39eb ]
~% echo $?
0
```

LGTM.

File: Formula/parity.rb (hunk 1 above)
Comment (mike-burns) [inline Formula/parity.rb]: `master` is the same as release?

Comment (geoffharcourt) [inline Formula/parity.rb]: At the moment, yes. The URL would point to a snapshot in time, linked to a commit SHA though.

**Pull request merged**